dockstring_example.ipynb

```diff
@@ -0,0 +1,314 @@
+            "Processing triggers for man-db (2.9.1-1) ...\n",
+            "Processing triggers for mime-support (3.64ubuntu1) ...\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 37,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "bRDMv3v9Z13j",
+        "outputId": "6da33955-d280-4f56-cbec-8322ebc1cc37"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "--2023-06-05 06:27:50--  https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh\n",
+            "Resolving repo.anaconda.com (repo.anaconda.com)... 104.16.130.3, 104.16.131.3, 2606:4700::6810:8303, ...\n",
+            "Connecting to repo.anaconda.com (repo.anaconda.com)|104.16.130.3|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 73134376 (70M) [application/x-sh]\n",
+            "Saving to: ‘Miniconda3-latest-Linux-x86_64.sh.1’\n",
+            "\n",
+            "Miniconda3-latest-L 100%[===================>]  69.75M   206MB/s    in 0.3s    \n",
+            "\n",
+            "2023-06-05 06:27:50 (206 MB/s) - ‘Miniconda3-latest-Linux-x86_64.sh.1’ saved [73134376/73134376]\n",
+            "\n",
+            "PREFIX=/usr/local\n",
+            "Unpacking payload ...\n",
+            "                                                                                  \n",
+            "Installing base environment...\n",
+            "\n",
+            "\n",
+            "Downloading and Extracting Packages\n",
+            "\n",
+            "\n",
+            "Downloading and Extracting Packages\n",
+            "\n",
+            "Preparing transaction: - \b\bdone\n",
+            "Executing transaction: | \b\b/ \b\b- \b\b\\ \b\bdone\n",
+            "installation finished.\n",
+            "WARNING:\n",
+            "    You currently have a PYTHONPATH environment variable set. This may cause\n",
+            "    unexpected behavior when running the Python interpreter in Miniconda3.\n",
+            "    For best results, please verify that your PYTHONPATH only points to\n",
+            "    directories of packages that are compatible with the Python interpreter\n",
+            "    in Miniconda3: /usr/local\n"
+          ]
+        }
+      ],
+      "source": [
+        "!wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh\n",
+        "!chmod +x Miniconda3-latest-Linux-x86_64.sh\n",
+        "!./Miniconda3-latest-Linux-x86_64.sh -b -f -p /usr/local"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 38,
+      "metadata": {
+        "id": "Zrnfcx3OZ7IP"
+      },
+      "outputs": [],
+      "source": [
+        "import sys\n",
+        "sys.path.append('/usr/local/lib/python3.7/site-packages')"
```

Comment:
Urmm to link the installed Python3.7 to the collab, but it's not working and I only did it to install RDKit, where I would love some help because the specific version of RDKit is not wanting to be installed with the version of Python Google Colab has as default